platform-sdk/swirlds-jasperdb/src/main/resources/com/swirlds/merkledb/files/hash_record.proto

```diff
@@ -8,7 +8,7 @@ option java_multiple_files = true;
 message HashRecord {
 
     // Virtual node path
-    optional uint64 path = 1;
+    optional fixed64 path = 1;
```

Comment:
It may not be immediately clear whether using a fixed-length number necessarily improves things w/o knowing the distribution of path values. But that's a minor issue per se.

A major issue here is that this changes the format of the data on the wire (aka in the serialized form), while the PR description claims that no data migration is necessary. If the latter is true, is this protobuf model used nowhere then? If so, why do we have it here? If it is used though, and the type of the field changes, then some data migration may be necessary, no?

The same comment applies to the leaf_record.proto just below.

Reply:
The field has always been `fixed64`. This change is to fix the documentation. No data migration is needed.

Reply:
I see. It's odd to see a file that seemingly contains real code (~ data definition) which isn't actually a compilable code, but instead is a piece of documentation. I'm unsure why we'd maintain it in a .proto file instead of putting it to javadoc or similar. But it's good to know there's no bugs. Thanks.